src/Symfony/Component/Form/Test/TypeTestCase.php

```diff
@@ -26,12 +28,27 @@
      */
     protected $dispatcher;
 
+    /**
+     * @var ValidatorInterface
+     */
+    private $validator;
+
     protected function setUp()
     {
         parent::setUp();
 
         $this->dispatcher = $this->getMockBuilder('Symfony\Component\EventDispatcher\EventDispatcherInterface')->getMock();
         $this->builder = new FormBuilder(null, null, $this->dispatcher, $this->factory);
+        $this->validator = $this->getMockBuilder('Symfony\Component\Validator\Validator\ValidatorInterface')->getMock();
```

Comment:
This will break existing applications that use the `TypeTestCase` class, but do not have the Validator component installed.